repo.go

```diff
@@ -477,13 +524,150 @@ func (r *Repo) RevokeKeyWithExpires(keyRole, id string, expires time.Time) error
 		root.Version++
 	}
 
-	err = r.setTopLevelMeta("root.json", root)
+	err = r.setMeta("root.json", root)
 	if err == nil {
 		fmt.Println("Revoked", keyRole, "key with ID", id, "in root metadata")
 	}
 	return err
 }
 
+// AddTargetsDelegation is equivalent to AddTargetsDelegationWithExpires, but
+// with a default expiration time.
+func (r *Repo) AddTargetsDelegation(delegator string, delegatedRole data.DelegatedRole, keys []*data.PublicKey) error {
```

Comment:
nite: Make the method more readable. 
```suggestion
func (r *Repo) AddDelegatedRole(delegator string, delegatee data.DelegatedRole, keys []*data.PublicKey) error {
```

Reply:
I changed the method name in https://github.com/theupdateframework/go-tuf/pull/175/commits/0f60bf5964080d24bd5ad7c9847b87aaca6f4d9f, but left the `delegatedRole` name, since to me the parallel names `delegator` and `delegatee` look like they should be the same type, but they are in fact different types.